common/src/main/java/io/netty/util/concurrent/SingleThreadEventExecutor.java

```diff
@@ -662,6 +680,29 @@ public void execute(Runnable task) {
         }
     }
 
+    /**
+     * Returns the {@link Thread} that powers the {@link SingleThreadEventExecutor}.
```

Comment:
It is worth adding a comment that this may block until this class finishes initializing?